README.adoc

```diff
@@ -2523,7 +2523,14 @@ validates :name, presence: true
 
 ==== Methods That Have "keyword" Status in Ruby [[method-invocation-parens-keyword]]
 
-Always omit parentheses for methods that have "keyword" status in Ruby.
+The set of methods that have "keyword" status is not well-defined.
```

Comment:
I think we still have to provide a guideline here (even if we soften the language) and add the extra information in some admonition block (e.g. NOTE:).

Reply:
OK, I force-pushed an amended commit that provides a guideline and adds extra info in an admonition block.